### ISSUE TITLE
netkan.exe should provide better diagnostics on license failure

### ISSUE THREAD

**Issue by pjf** [KSP-CKAN/CKAN#1469]
Right now, `netkan.exe` provides a cryptic message if we index a mod
from KerbalStuff and can't understand the license:

```
$ netkan.exe Deltaglider.netkan 
2979 [1] FATAL CKAN.NetKAN.Program (null) - Exception has been thrown by the target of an invocation.
```

I believe that this has been masked because on pull requests our test suite explicitly checks to see if there's a license field, or an `x_netkan` field assuring us that the license will be okay. However when manually working with files, the error can sure be confusing!

Desired behaviour: We provide a better error message when the license (or other fields) provide validation as part of the pipeline.

**Comment by plague006** [comment]
+1

**Comment by plague006** [comment]
It's a 2-star in my heart, but a 1-star in reality.

**Comment by politas** [comment]
:+1:

**Comment by politas** [comment]
Maybe we could also have a special test for: Noun 'licence' used instead of verb 'license'!
Just to help out those of us whose native English has not undergone spelling reformation.

**Comment by HebaruSan** [comment]
I believe this was fixed in #2480; we now discard wrapper exceptions and print out the `InnerException`. Example with locally modified `KSP-AVC.netkan`:

```
$ ../CKAN/_build/netkan.exe NetKAN/KSP-AVC.netkan 
992 [1] FATAL CKAN.NetKAN.Program (null) - The license GPL-4.0 is invalid
```
